toolbox/Dockerfile

```diff
@@ -15,7 +15,6 @@
 FROM ubuntu:xenial
 MAINTAINER Jared Watts <jared.watts@quantum.com>
 
-
 RUN apt-get update && \
```

Comment:
good stuff @bzub, i'll have some time this weekend to try out the `toolbox` with the `ceph` tools for `kraken`.  I definitely agree that having matching client:server is always a good idea.  thanks for the input!